src/zc/relation/tokens.txt

```diff
@@ -106,23 +195,26 @@ to sanely and transparently hold a set of persistent objects, try the
 zc.set package XXX not yet.)
 
     >>> import BTrees
-    >>> class Organization(persistent.Persistent):
-    ...     zope.interface.implements(IOrganization)
+    >>> import persistent
+    >>> @zope.interface.implementer(IOrganization)
+    ... class Organization(persistent.Persistent):
+    ...
     ...     def __init__(self, title):
     ...         self.title = title
     ...         self.parts = BTrees.family32.IF.TreeSet()
     ...     # the next parts just make the tests prettier
     ...     def __repr__(self):
     ...         return '<Organization instance "' + self.title + '">'
-    ...     def __cmp__(self, other):
+    ...     def __lt__(self, other):
     ...         # pukes if other doesn't have name
-    ...         return cmp(self.title, other.title)
+    ...         return self.title < other.title
```

Comment:
I don't like seeing only one of six rich comparison methods being defined. If you use @functools.total_ordering, you can get away by defining two of six.